kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -64,18 +64,27 @@
               {{ formattedCoachNamesTooltip(row[3]) }}
             </KTooltip>
           </span>
-          <span v-else-if="colIndex === 2">
+          <span
+            v-else-if="colIndex === 2"
+            style="display: flex; justify-content: start"
+          >
             {{ content }}
           </span>
           <span
             v-else-if="colIndex === 3"
             class="core-table-button-col"
           >
-            <KButton
-              appearance="flat-button"
-              :text="$tr('deleteClass')"
-              @click="selectClassToDelete(row[3])"
-            />
+            <KIconButton icon="optionsVertical">
+              <template #menu>
+                <KDropdownMenu
+                  :primary="false"
```

Comment:
(Blocking) KDropdownMenu doesnt receive neither the `primary` nor `disabled` props.

`hasIcons` should not be true, as we are not rendernig any icon in this dropdown